src/api/call-api.test.ts

```diff
@@ -188,58 +65,54 @@ describe('callApi', () => {
     }));
 
     const [logs, res] = await callApi({
-      ...callApiOptions,
-      oises: oisesWithoutType,
+      config: { ...config, ois: oisesWithoutType },
+      aggregatedApiCall,
     });
 
-    expect(logs).toHaveLength(1);
+    expect(logs).toHaveLength(2);
     expect(logs).toEqual(
       expect.arrayContaining([
         {
           level: 'ERROR',
-          message: expect.stringMatching("reserved parameter '_type' is missing for endpoint: convertToUSD"),
+          message: "Cannot read property 'length' of undefined",
+        },
+        {
+          error: "Cannot read property 'length' of undefined",
+          level: 'ERROR',
+          message:
```

Comment:
So due to the double logs from `callApi` these are still incorrect, 

Should we just go with the default (error) logs returned by` node.handlers.callApi`, or use the more detailed logs @acenolaza you had before. Mainly, we no longer have access to the `apiResponse` values :thinking:

Reply:
This is now testing node.callApi() internals so maybe that should be tested in the node package but I'm surprised to see that this check has been removed from the [buildOptions()](https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/api/index.ts#L13) function.

Reply:
hmm what kind of testing should we do here instead? just `callApi` with mocked success and error cases?

Reply:
Right, this function is now much simpler so I guess most tests we used to have might no longer apply